src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -72,11 +70,11 @@ abstract class CompilerFleetMessageReceiver : MessageReceiver {
 private val LOGGER = LoggerFactory.getLogger("TaskReceiver")
 
 class TaskReceiver(tasksDirectory: String,
-                   private val resultPublisher: Publisher,
-                   private val callback: (message: String, filename: String) -> Unit
+                   private val resultPublisher: Publisher
 ) : CompilerFleetMessageReceiver() {
-    private val dockerProcessor = DockerProcessor(getDefaultDockerClient())
     private val tasksDir: Path
+    private val dockerProcessor = DockerProcessor(getDefaultDockerClient())
+    private val MOCK_PDF_BYTES = TaskReceiver::class.java.classLoader.getResourceAsStream("example.pdf").readBytes()
```

Comment:
As faras I remember, there is `getResourceAsStream` directly in `Class` class, no need to call classLoader

Reply:
Indeed, method `getResourceAsStream` in class exists, but I've got some strange error when launching main:
```
Exception in thread "main" java.lang.IllegalStateException: TaskReceiver::class.java…ceAsStream("example.pdf") must not be null
	at com.bardsoftware.backend.fleet.rmarkdown.TaskReceiver.<init>(Subscriber.kt:77)
	at com.bardsoftware.backend.fleet.rmarkdown.SubscriberKt.main(Subscriber.kt:180)

```

So I leave it untouched.